concent-vm/helper-scripts/concent-update.sh

```diff
@@ -0,0 +1,24 @@
+#! /bin/bash -e
+
+DEFAULT_BRANCH=master
+
+branch_name="${1:-$DEFAULT_BRANCH}"
+
+cd ~/concent/concent_api/
+
+# Fetch new code and checkout specific branch
+git fetch --prune
```

Comment:
You should always specify the remote when doing `git fetch` otherwise the behavior can be different depending on what's in `.gitconfig` (it could fetch from the default remote or from all remotes; the default remote can also differ).

Reply:
Done.